Assets/Scenes/Lobby/ServerInfo.gd

```diff
@@ -0,0 +1,35 @@
+extends Control
+
+var selected_server: Control = null
+
+func _update_info():
+	if selected_server.response != null:
+		$VBoxContainer/RichTextLabel.text = selected_server.response.get("motd", "no motd")
+		$VBoxContainer/HBoxContainer2/Join.disabled = false
+	else:
+		$VBoxContainer/HBoxContainer2/Join.disabled = true
+	pass
+
+func _on_server_list_server_selected(server):
+	selected_server = server
+	_update_info()
+	pass # Replace with function body.
+
+
+func _on_server_list_server_updated(server):
+	if selected_server == server:
+		_update_info()
+	pass # Replace with function body.
+
+func _get_model_number(model: String):
```

Comment:
why are you doing it this way instead of just sending the actual string? this would make things less modular

Reply:
will merge and fix